### PR TITLE
Update requirements to include qualtrics XBlock

### PR DESCRIPTION
**Pull request by dcadams**
Update requirements to include qualtrics XBlock and small tidyup.

### CODE REVIEW

**Comment by dcadams** [comment]
@stvstnfrd  here's a quick one.

File: requirements/edx/github.txt [hunk 1]
```diff
@@ -50,8 +50,6 @@ git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c
 -e git+https://github.com/mckinseyacademy/xblock-poll.git@09e8feda40fdf77bd19b830b497ff969891e11af#egg=xblock-poll
 -e git+https://github.com/Stanford-Online/xblock-mufi.git@ee853b8a7668a87c27d13d55c0d149a04b8657b8#egg=xblock_mufi-master
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
-#-e git+https://github.com/edx/edx-ora2.git@release-2015-05-08T16.15#egg=edx-ora2
-#-e git+https://github.com/edx/edx-submissions.git@e2361932b9bce061a018a31bb3929e9cade80f49#egg=edx-submissions
```
Comment (stvstnfrd) [inline requirements/edx/github.txt]: ?

Comment (dcadams) [inline requirements/edx/github.txt]: It's a relic from a long time ago; just noticed it.

Comment (stvstnfrd) [inline requirements/edx/github.txt]: Ah, gotcha; I didn't see the comment at first.

Can we split this into 2 commits then?

```shell
git reset HEAD^
git add --patch
# y
# n
git commit -m "Removed commented-out requirements"
git add --update
git commit -m "Install QualtricsXBlock via requirements.txt"
```

**Comment by stvstnfrd** [comment]
See comment about splitting commits, otherwise +1

**Comment by stvstnfrd** [comment]
Thanks @dcadams !

**Pull request merged**